plugins/chat/assets/javascripts/discourse/models/chat-thread.js

```diff
@@ -28,6 +32,17 @@ export default class ChatThread {
     // only has a small subset of message stuff.
     this.originalMessage = args.original_message;
     this.originalMessage.user = this.originalMessageUser;
+
+    // TODO (martin) Drafts per thread?
+    // if (this.currentUser.chat_drafts) {
+    //   const storedDraft = this.currentUser.chat_drafts.find(
+    //     (draft) => draft.channel_id === this.channel_id
+    //   );
+    //   this.draft = ChatMessageDraft.create(
+    //     storedDraft ? JSON.parse(storedDraft.data) : null
+    //   );
+    // }
+    this.draft = ChatMessageDraft.create(null);
```

Comment:
I would say we don't bother with drafts in threads for now

Reply:
Yeah I thought that may be a bit advanced, will remove